### PR TITLE
lesson 1.7

### PR DESCRIPTION
**Pull request by CHEUNG-K-Jeffrey**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -28,7 +41,7 @@ const App = () => {
     <>
       <h1>Todo List</h1>
       <AddTodoForm onAddTodo={addTodo} />
-      <p></p>
+      {isLoading ? <p>Loading list...</p> : null}
       <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
```
Comment (justiceotuya) [inline src/App.jsx:45]: welldone Jeffery, i just need you to make a simole change, the tenary `isLoading` statement is used to show the `loading list...` text when the app is load and waiting for data or show the `Todolist` when the data is present so I suggest you do this

```suggestion
{isLoading ? <p>Loading list...</p> : 
      <TodoList todoList={todoList} onRemoveTodo={removeTodo} /> }
```

**Review by justiceotuya — CHANGES_REQUESTED** [review]
Please implement the suggested changes

File: src/App.jsx [hunk 2]
```diff
@@ -28,7 +41,7 @@ const App = () => {
     <>
       <h1>Todo List</h1>
       <AddTodoForm onAddTodo={addTodo} />
-      <p></p>
+      {isLoading ? <p>Loading list...</p> : null}
```
Comment (CHEUNG-K-Jeffrey) [inline src/App.jsx:44]:
```suggestion
```

**Review by CHEUNG-K-Jeffrey — COMMENTED** [review]
Remove isLoading logic

**Review by justiceotuya — APPROVED** [review]
Welldone

**Pull request merged**